terraform/modules/emr/templates/emr/configuration.mysql.json

```diff
@@ -51,6 +51,15 @@
       "javax.jdo.option.ConnectionPassword": "${hive_metastore_pwd}"
     }
   },
+  {
```

Comment:
Do we need to remove {hive_metastore_database_name} from spark configs?

Reply:
it needs to be there for analytical-env to know which database to connect to